src/js/components/Panel/Panel.stories.js

```diff
@@ -0,0 +1,102 @@
+
+ const imageOptions = {
+  'None': noImage,
+  'Bill Murray (640x360)': billMurrayImage,
+  'Cats (800x300)': catImage,
+  'Random (300x300)': randomImage,
+  'Animal (500x200)': animalImage,
+  'Brown Bear (1000x750)': brownBearImage,
+  'Beards (100x150)': beardImage,
+  'Architecture (300x800)': architectureImage
+ };
+
+ stories.add('Default', (() =>
+  <Panel mods='u-padMd'>
+    Base Panel with padding utility
+  </Panel>
+))
+
+stories.add('Full Panel', (() =>
+  <Panel>
+      <PanelHeader>
+        <h3 className='Panel-title'>Full Panel, with Nested Child Elements</h3>
+      </PanelHeader>
+      <PanelBody>
+        <PanelRow mods='Grid'>
+          <PanelCell mods='Grid-cell u-size'>
+            <b>Bill Murray is an American actor...</b>
+            <p>
+            comedian, and writer. The fifth of nine children, he was born William James 
+            Murray in Wilmette, Illinois, to Lucille (Collins), a mailroom clerk, and Edward Joseph Murray II, who sold 
+            lumber. He is of Irish descent. Among his siblings are actors Brian Doyle-Murray, Joel Murray, and John Murray. 
+            He and most of his siblings worked as caddies, which paid his tuition to Loyola Academy, a Jesuit school. He played 
+            sports and did some acting while in that school, but in his words, mostly "screwed off." He enrolled at Regis College 
+            in Denver to study pre-med but dropped out after being arrested for marijuana possession. He then joined the National 
+            Lampoon Radio Hour with fellow members Dan Aykroyd, Gilda Radner, and John Belushi. However, while those three became 
+            the original members of Saturday Night Live (1975), he joined Saturday Night Live with Howard Cosell (1975), which 
+            premiered that same year. After that show failed, he later got the opportunity to join Saturday Night Live (1975), 
+            for which he earned his first Emmy Award for Outstanding Writing in a Comedy-Variety or Music Series. He later went 
+            on to star in comedy films. He also co-directed Quick Change (1990). Murray garnered additional critical acclaim later 
+            in his career, starring in Lost in Translation (2003), which earned him a Golden Globe and a BAFTA Award for Best Actor, 
+            as well as an Academy Award nomination for Best Actor. He also received Golden Globe nominations for his roles in 
+            Ghostbusters, Rushmore (1998), Hyde Park on Hudson (2012), St. Vincent (2014), and the HBO miniseries Olive Kitteridge 
+            (2014), for which he later won his second Primetime Emmy Award for Outstanding Supporting Actor in a Limited Series or a Movie.
+            </p>
+          </PanelCell>
+          <PanelCell mods='Grid-cell'>
+              <img src='https://www.fillmurray.com/640/360' />
+          </PanelCell>
```

Comment:
is there a reason you aren't using panelImage here?

Reply:
Yeah an image in a panel is different from a "panel image". I updated it to headerImage in the next commit to make it a little clearer.

<img width="484" alt="Screen Shot 2019-06-05 at 1 46 11 PM" src="https://user-images.githubusercontent.com/1189536/58985330-51e12880-8798-11e9-976d-4dd7622d3e37.png">
<img width="1410" alt="Screen Shot 2019-06-05 at 1 46 18 PM" src="https://user-images.githubusercontent.com/1189536/58985331-51e12880-8798-11e9-9c86-9d736a5e7260.png">

On it's just content, the other the image become more a part of the panel